src/Components/AppShell/AppShell.scss

```diff
@@ -7,7 +7,7 @@
     display: flex;
     flex-grow: 1;
     justify-content: center;
-    overflow: scroll;
+    overflow: hidden;
```

Comment:
might be worth a comment here just to say that scrolling is handled by a react component